ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.14.x__fuse-online-ocp4.14-lp-interop.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: docker
```

Comment:
It looks like[ this Dockerfile](https://github.com/syndesisio/syndesis-qe/blob/1.14.x/docker/Dockerfile#L121) ends using entrypoint, when the pod is built I believe that the entrypoint.sh is not being executed.

[A thread ](https://redhat-internal.slack.com/archives/CBN38N3MW/p1682403979742489) from the DPTP basically says that it is ignored.

I would suggest moving anything that is done in the[ entrypoint.sh script](https://github.com/syndesisio/syndesis-qe/blob/1.14.x/docker/entrypoint.sh) and move that to the commands.sh file within the ref to be sure that whatever is supposed to be happening there is actually happening.